src/NServiceBus.RabbitMQ/Administration/QueueCreator.cs

```diff
@@ -50,5 +53,46 @@ void CreateQueueIfNecessary(IModel channel, string receivingAddress)
 
             routingTopology.Initialize(channel, receivingAddress);
         }
+
+        void BuildDeferralInfrastructure(IModel channel)
+        {
+            const int maxLevel = 32 - 1;
```

Comment:
Do we want to expose this value somewhere to be configurable? Or just set it to 32 for now?